clients/drcachesim/CMakeLists.txt

```diff
@@ -499,6 +499,17 @@ endif ()
 # Be sure to give the targets qualified test names ("tool.drcache*...").
 
 if (BUILD_TESTS)
+  add_executable(tool.drcachesim.cache_lru_unit_tests tests/cache_lru_unit_tests.cpp)
```

Comment:
I would vote for not having a separate executable: a separate file makes sense but we can link all the unit tests together into one executable.  I'm not sure I see any advantage of separate executables: we're always going to run all of them.  There are a number of downsides: extra build code complexity; extra build time and extra test time increasing the CI time.

Reply:
Done.